### PR TITLE
Fixed issue #11 - server_names on new lines missed

### PR DESCRIPTION
**Pull request by LukeShirnia**
This PR fixes issue #11, small code updated vs PR #8.
New line directives for server_name should now be listed in nginxctl ouput.

### CODE REVIEW

File: nginxctl.py [hunk 1]
```diff
@@ -387,6 +387,7 @@ def _get_vhosts_info(self, config_file):
             ip_port = []
             server_name_found = False
             server_dict = {}
+            stored = ''
```
Comment (LukeHandle) [inline nginxctl.py]: Another name for this variable might help better explain its use? eg. `multi_line_buffer` (still not great)

File: nginxctl.py [hunk 2]
```diff
@@ -401,7 +402,14 @@ def _get_vhosts_info(self, config_file):
                 if l.startswith('#'):
                     continue
                 l = l.split('#')[0]
+
+                if not l.strip().endswith(';'):
+                    if line_num != server_block[0]:
+                        stored += l.strip() + ' '
+                    continue
+                l = stored + l
```
Comment (LukeHandle) [inline nginxctl.py]: Though this is not "wrong", would it be cleaner to wrap it in an `if`?

```
if stored:
    l = stored + l
    stored = ''
```

Is this clearer or not?

**Comment by LukeHandle** [comment]
Can you also please update the tests file to include an example of a multi-line directive?

File: nginxctl.py (hunk 1 above)
Comment (LukeShirnia) [reply nginxctl.py]: Renamed to `store_multiline`

**Pull request merged**